x/evm/abci.go

```
@@ -699,6 +698,12 @@ func handleMessages(ctx sdk.Context, bk types.BaseKeeper, n types.Nexus, m types
 				continue
 			}
 
+			if srcCk, err := bk.ForChain(ctx, msg.GetSourceChain()); err == nil {
```

can we use a `Must` so we panic if `err` is not `nil` since that's unexpected?